decidim-core/lib/decidim/core.rb

```diff
@@ -93,6 +96,13 @@ def self.seed!
     %w(en ca es eu fi fr gl it nl pt pr-BR ru sv uk)
   end
 
+  # Exposes a configuration option: an array of symbols that matches a ContentParser, and
+  # their equivalent ContentRenderer, that will be automatically executed when the content
+  # is parsed or rendered respectively.
+  config_accessor :content_processors do
```

Comment:
Shouldn't this array should contain the `UserProcessor`? Also, missing docs on the expected syntax: how is this array supposed to be formatted? Is this an array of class names?

Reply:
The `user` processor is registered in an initializer declared in the engine, as the `abilities`:

```
initializer "decidim.content_processors" do |_app|
  Decidim.configure do |config|
    config.content_processors += [:user]
  end
end
```

It's ok? Or better remove the initializer and add the processor directly here?

I'll add some examples of the expected syntax.